python/kserve/test/test_server.py

```diff
@@ -1217,21 +1133,16 @@ def test_predict_ce_avro_binary(self, http_server_client):
 
 
 class TestTFHttpServerLoadAndUnLoad:
-    @pytest.fixture(scope="class")
-    def app(self):
-        server = ModelServer(
-            registered_models=DummyModelRepository(test_load_success=True)
-        )
-        rest_server = RESTServer(
-            kserve_app, server.dataplane, server.model_repository_extension
+    @pytest_asyncio.fixture(scope="class", autouse=True)
+    def app(self, server):
+        mp = pytest.MonkeyPatch()
```

Comment:
instead of Monkey patch, we can create a different server fixture?

Reply:
The fastapi app is global variable. Registering endpoints and middlewares again will lead to runtime errors.